src/components/community/Platforms/platformsData.ts

```diff
@@ -0,0 +1,34 @@
+import { DiscordLogo, TelegramLogo } from "phosphor-react"
+
+const platformsData = {
+  telegram: {
+    logo: TelegramLogo,
+    join: {
+      title: "Join Telegram group",
```

Comment:
Should be just "Join Telegram", there're multiple groups and a bot chat. At every occurrence below too